src/main/scala/Comparator.scala

```diff
@@ -0,0 +1,38 @@
+package comparator
+
+/** Bundle of comparison functions
+  */
+class Comparator {
+
+  /** Get Jaccard similarity between two sorted sets
+   * @param s1 First sorted set
+   * @param s2 Second sorted set
+   * @return Jaccard similarity (s1, s2)
+   */
+  def getJaccardSim(s1 : scala.collection.SortedSet[Int], s2 : scala.collection.SortedSet[Int]) : Float = {
+    val intersection = s1.intersect(s2)
+    val union = s1.union(s2)
+    return intersection.size.toFloat/union.size.toFloat
```

Comment:
Return statements in Scala are not idiomatic!

Reply:
Meaning I shouldn't write return?

Reply:
exactly. Last object in each function will be returned